src/sequence_state.cc

```diff
@@ -146,15 +199,25 @@ SequenceStates::Initialize(
         state_size =
             triton::common::GetByteSize(state.second.data_type(), dims);
       }
-      data = std::make_shared<AllocatedMemory>(
-          state_size, TRITONSERVER_MEMORY_CPU, 0);
+      if (use_growable_memory) {
+        // TODO: use a constant or a flag for virtual address space size.
+        std::unique_ptr<GrowableMemory> growable_memory;
+        RETURN_IF_ERROR(GrowableMemory::Create(
+            state_size, TRITONSERVER_MEMORY_GPU, device_id, growable_memory,
+            1024 * 1024 * 1024));
```

Comment:
If we stick with a constant default and not expose it as a flag for now, we should document it somewhere too, such as the model_config.proto in common or something.

Reply:
I added a flag.